### PR TITLE
Add Coffee Hispanic Tournament article

### PR DESCRIPTION
**Pull request by fajar13k**
Start of Coffee Hispanic Tournament (EN/ES) (Tournament has ended)

### CODE REVIEW

File: wiki/Tournaments/en.md [hunk 1]
```diff
@@ -154,6 +154,12 @@ Unofficial tournaments/competitions hosted by the communities.
 | :-- | :-- | :-- | :-- | :-- | :-- |
 | [osu! Asia Oceania Tournament](o!AOT/1) | 2019-07-26 | 2019-09-22 | 進撃のバブルティー | 黃建智 | 과로사 |
 
+### ![](/wiki/shared/mode/osu.png) Coffee Hispanic Tournament
+
+| Name | Start | End | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] |
+| :-- | :-- | :-- | :-- | :-- | :-- |
+| [Coffee Hispanic Tournament](CHT/1) | 2020-01-17 | 2019-02-17 | United States | Noveno Gaston | hallo y caspi |
```
Comment (cl8n) [inline wiki/Tournaments/en.md]:
```suggestion
| [Coffee Hispanic Tournament](CHT/1) | 2020-01-17 | 2020-02-17 | United States | Noveno Gaston | hallo y caspi |
```

File: wiki/Tournaments/CHT/1/en.md [hunk 2]
```diff
@@ -0,0 +1,476 @@
+---
+tags:
+  - CHT
+---
+
+# Coffee Hispanic Tournament
+
+![CHT logo](img/logo.jpg)
+
+The **Coffee Hispanic Tournament** (***CHT***) was a hispanic single-elimination 2v2 osu! tournament hosted by ![][flag_AR] [Vaf](https://osu.ppy.sh/users/12589048), ![][flag_AR] [iNeuu\_Z](https://osu.ppy.sh/users/9112463), and ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811). This was the first installment of the Coffee Hispanic Tournament.
```
Comment (cl8n) [inline wiki/Tournaments/CHT/1/en.md]: unclear what the qualifier "hispanic" means here, were you required to speak spanish to enter? or restricted to some countries?

File: wiki/Tournaments/CHT/1/es.md [hunk 3]
```diff
@@ -0,0 +1,476 @@
...
+
+| Equipo 1 |  |  | Equipo 2 | Link del match |
+| --: | :-: | :-: | :-- | :-- |
+| **hallo y caspi** | **5** | 2 | Radical | [#1](https://osu.ppy.sh/community/matches/58108351) |
+
+Domingo, 2 Febrero 2020:
+
+| Equipo 1 |  |  | Equipo 2 | Link del match |
+| --: | :-: | :-: | :-- | :-- |
+| **Noveno Gaston** | **0** | -1 | Geometry Dash | *win by default* |
```
Comment (cl8n) [inline wiki/Tournaments/CHT/1/es.md]: "win by default" doesnt have translation ?

File: wiki/Tournaments/CHT/1/en.md (hunk 2 above)
Comment (fajar13k) [reply wiki/Tournaments/CHT/1/en.md]: I think host meant countries that speak spanish, General Rules no. 3 if you want to check

Comment (cl8n) [reply wiki/Tournaments/CHT/1/en.md]: could use some clarification here then, I think you should remove hispanic from first sentence (it's already in the title) and add one more sentence after that saying it's country restricted

Comment (fajar13k) [reply wiki/Tournaments/CHT/1/en.md]:
```suggestion
The **Coffee Hispanic Tournament** (***CHT***) was a country-restricted single-elimination 2v2 osu! tournament hosted by ![][flag_AR] [Vaf](https://osu.ppy.sh/users/12589048), ![][flag_AR] [iNeuu\_Z](https://osu.ppy.sh/users/9112463), and ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811). This was the first installment of the Coffee Hispanic Tournament.
```
Rather this?

Comment (TPGPL) [reply wiki/Tournaments/CHT/1/en.md]:
```suggestion
The **Coffee Hispanic Tournament** (***CHT***) was a single-elimination 2v2 osu! tournament hosted by ![][flag_AR] [Vaf](https://osu.ppy.sh/users/12589048), ![][flag_AR] [iNeuu\_Z](https://osu.ppy.sh/users/9112463), and ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811). The tournament was restricted to Spanish-speaking countries only. This was the first installment of the Coffee Hispanic Tournament.
```
or something like that

File: wiki/Tournaments/CHT/1/en.md [hunk 4]
```diff
@@ -0,0 +1,476 @@
+---
+tags:
+  - CHT
+---
+
+# Coffee Hispanic Tournament
+
+![CHT logo](img/logo.jpg)
```
Comment (TPGPL) [inline wiki/Tournaments/CHT/1/en.md]:
```suggestion
![CHT Logo](img/logo.jpg)
```

Comment (fajar13k) [reply wiki/Tournaments/CHT/1/en.md]: Isn't this suppose to be sentence cased too?

Comment (TPGPL) [reply wiki/Tournaments/CHT/1/en.md]: nevermind then, can be left as it is now

File: wiki/Tournaments/CHT/1/es.md (hunk 3 above)
Comment (fajar13k) [reply wiki/Tournaments/CHT/1/es.md]: It has but it doesn't really fit, they told me it's better to leave it as it is.

File: wiki/Tournaments/CHT/1/en.md (hunk 4 above)
Comment (fajar13k) [reply wiki/Tournaments/CHT/1/en.md]: nvm, after some consulting we decide to translate it ✌

**Pull request merged**